examples/wCCD/src/wCCD.tsx

```diff
@@ -133,10 +133,23 @@ export default function wCCD(props: WalletConnectionProps) {
     const [admin, setAdmin] = useState<string>();
     const [adminError, setAdminError] = useState('');
 
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const testnet = 'testnet';
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const mainnet = 'mainnet';
```

Comment:
```suggestion
```
Or why are you keeping them? :thinking:

Reply:
`process.env.NETWORK` is replaced by e.g. testnet (not by the string `testnet`) and without defining testnet=`testnet`, I got an error that testnet variable is not defined.